examples/jaeger-hotrod/docker-compose.yml

```diff
@@ -54,11 +51,14 @@ services:
       - my_network
   opensearch:
     container_name: node-0.example.com
-    image: opensearchproject/opensearch:2.9.0
+    image: opensearchproject/opensearch:2.16.0
     environment:
       - discovery.type=single-node
       - bootstrap.memory_lock=true # along with the memlock settings below, disables swapping
       - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # minimum and maximum Java heap size, recommend setting both to 50% of system RAM
+      - OPENSEARCH_INITIAL_ADMIN_PASSWORD=opensearchNode1Test
```

Comment:
We should have a common password for our demos to make it easier for the community to move between them. Perhaps `yourStrongPassword123!` as this is what the [blog post on the new password](https://opensearch.org/blog/replacing-default-admin-credentials/) used.

Also, can you update the `README.md` to include the password?

https://github.com/opensearch-project/data-prepper/blob/d643ed425cbca99f681b3378eb7b789e983616f5/examples/jaeger-hotrod/README.md?plain=1#L18